src/main/java/frc/robot/commands/pretest/MotorTestCommand.java

```diff
@@ -60,7 +62,7 @@ public MotorTestCommand(BaseSwerveSubsystem swerveSubsystem, TiltedElevatorSubsy
 
             // Test dropper command from `CONE_MID`
             DropperChooserCommand.getSequence(swerveSubsystem, rollerSubsystem, tiltedElevatorSubsystem),
```

Comment:
We deleted swerve because we wanted to run it on the cart, but the dropper command still runs swerve to back up doesn't it? Is the small duration it runs the wheels for ok?

Reply:
that's a good point... idk 😓